Project.toml

```diff
@@ -18,10 +18,20 @@ StaticArraysChainRulesCoreExt = "ChainRulesCore"
 StaticArraysStatisticsExt = "Statistics"
 
 [compat]
-Aqua = "0.7"
+Aqua = "0.8"
+BenchmarkTools = "1"
 ChainRulesCore = "1"
+ChainRulesTestUtils = "1"
+InteractiveUtils = "1"
+JLArrays = "0.1"
+LinearAlgebra = "1"
```

Comment:
I usually do lower bounds for stdlibs that are the same as the version of Julia. I think this looks coherent because we don't support older stdlibs anyway.

Reply:
Sure, I'll change that. The caveat is that several compat bounds would need to be updated if there's a bump in the minimum Julia version that's supported.